man/fi_msg.3.md

```diff
@@ -234,7 +234,7 @@ fi_sendmsg.
 
 *FI_DISCARD*
 : Applies to posted receive operations for endpoints configured
-  for FI_BUFFERED_RECV or FI_VARIABLE_MSG.  This flag is used to
+  for FI_BUFFERED_RECV.  This flag is used to
```

Comment:
In addition to the `FI_VARIABLE_MSG` flag, the `FI_BUFFERED_RECV` flag was _also_ removed in this pull request. Therefore, shouldn't the `FI_CLAIM` and `FI_DISCARD` flags _also_ be removed?